gnd/src/main/res/navigation/nav_graph.xml

```diff
@@ -167,4 +173,20 @@
     android:id="@+id/showHomeScreen"
     app:destination="@id/home_screen_fragment"
     app:popUpTo="@id/navGraph" />
-</navigation>
\ No newline at end of file
+  <action
+    android:id="@+id/showTermsScreen"
+    app:destination="@id/terms_fragment"
+    app:popUpTo="@id/navGraph" />
+
+  <fragment
+    android:id="@+id/terms_fragment"
+    android:name="com.google.android.gnd.ui.terms.TermsOfServiceFragment"
+    android:label="terms_frag"
+    tools:layout="@layout/fragment_terms_service">
+    <action
+      android:id="@+id/proceedDirectlyToHomeScreen"
```

Comment:
Why "directly"? Should this be just "showHomeScreen"?

Reply:
changed proceedToHomeScreen  as showHomeScreen was also assigned to  updated in #845